install/tests/Makefile

```diff
@@ -226,7 +226,7 @@ endif
 
 KOTS_KONFIG := "./manifests/kots-config.yaml"
 
-get-base-config: get-github-config
+get-base-config:
```

Comment:
**question:** Is this change related to the described issue, or is this a general fixup?